extensions/jsonb/deployment/src/main/java/io/quarkus/jsonb/deployment/JsonbProcessor.java

```diff
@@ -47,4 +69,75 @@ public boolean test(BeanInfo bean) {
         }));
     }
 
+    // Generate a JsonbConfigCustomizer bean that registers each serializer / deserializer with JsonbConfig
+    @BuildStep
+    void generateCustomizer(BuildProducer<GeneratedBeanBuildItem> generatedBeans,
+            List<JsonbSerializerBuildItem> serializers,
+            List<JsonbDeserializerBuildItem> desrializers) {
+
+        if (serializers.isEmpty()) {
+            return;
+        }
+
+        final Set<String> customSerializerClasses = new HashSet<>();
+        final Set<String> customDeserializerClasses = new HashSet<>();
+        for (JsonbSerializerBuildItem serializer : serializers) {
+            customSerializerClasses.addAll(serializer.getSerializerClassNames());
+        }
+        for (JsonbDeserializerBuildItem deserializer : desrializers) {
```

Comment:
```suggestion
        for (JsonbDeserializerBuildItem deserializer : deserializers) {
```
(obviously, better do that with a refactor ;))

Reply:
I'm very surprised my IDE didn't warn me... I want my money back :)